ansible/roles/master/handlers/main.yml

```diff
@@ -14,12 +14,15 @@
 
 - name: restart apiserver
   service: name=kube-apiserver state=restarted
+  when: groups['masters']|length == 1
```

Comment:
I don't  follow. groups['masters'][0] is used elsewhere to delegate a task to just one master (generating tokens and certs is one example). In this case, we only want to restart the apiserver service when there is only one master.